src/content/docs/style-guide/structure/tiles.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Tiles
+tags:
+  - Basic style guide
+  - Style guide quick reference
+redirects:
+---
+
+import styleTileTypes from 'images/style-guide_screenshot-full_tile-examples.webp'
+
+DocTiles, or just Tiles for short, are a unique type of react component designed to provide some visual flair to groups of links. Laid out in a grid, tiles can present a lot of links on a condensed space without overwhelming the reader. These are most useful for docs which direct to numerous places as part of their user journey, such as intro docs.
```

Comment:
in keeping with our sentence case style guide, i wonder if we want to capitalize these things or just say 'tabs' or 'tiles' or whatever. in context probably fine to use sentence case

Reply:
Fixed!